couchbase/src/main/scala/akka/stream/alpakka/couchbase/CouchbaseSessionRegistry.scala

```diff
@@ -27,10 +27,30 @@ object CouchbaseSessionRegistry extends ExtensionId[CouchbaseSessionRegistry] wi
     new CouchbaseSessionRegistry(system)
 
   /**
-   * Java API: get the session registry
+   * Get the session registry with new actors API.
    */
-  override def get(system: ActorSystem): CouchbaseSessionRegistry =
-    super.get(system)
+  // This is not source compatible with Akka 2.6 as it lacks `overrride`
```

Comment:
I think you talked about that. So, that means it will still work on the bytecode level as long as alpakka is compiled against Akka 2.5 but needs changing once Akka 2.5 support is removed?

Reply:
Yes. The byte-code-level method signature is unaffected by the `override` so it works as expected with Akka 2.6.
But compilation with Akka 2.6 fails as the compiler requires the `override`.